app/assets/stylesheets/common/d-editor.scss

```diff
@@ -208,6 +208,28 @@
         cursor: pointer;
       }
     }
+
+    .alt-text-readonly-container {
+      display: flex;
+      background: var(--secondary);
+
+      span {
+        margin-right: 0.5em;
+        overflow: hidden;
+        white-space:nowrap;
+        text-overflow:ellipsis;
```

Comment:
I see its lacking spaces here, be sure to configure your editor with prettier/eslint to follow our code standards please

Reply:
*shakes fist* I trusted you webstorm!